src/components/navigation.js

```diff
@@ -62,38 +57,16 @@ export function Navigation() {
             <NavigationLink name="Resources" id="resources" />
             <NavigationLink name="Conclusion" id="conclusion" />
 
-            <div className="border-bg-menu-back-light mt-16 flex flex-col border-t py-16 md:mt-4 md:flex-row md:items-center md:justify-between md:p-8">
-              <div className="bg-logo mb-16 aspect-square h-48 bg-contain bg-center bg-no-repeat invert md:mb-0 md:h-16"></div>
-              <div className="flex justify-evenly gap-4 text-center text-5xl text-menu-back-dark md:text-3xl">
-                <a
-                  href="https://www.instagram.com/veganhacktivists"
-                  target="_blank"
-                  rel="noreferrer"
-                >
-                  <InstagramIcon />
-                </a>
-                <a
-                  href="https://www.youtube.com/c/VeganHacktivists"
-                  target="_blank"
-                  rel="noreferrer"
-                >
-                  <YoutubeIcon className="h-28 w-28" />
-                </a>
-                <a
-                  href="https://www.patreon.com/veganhacktivists"
-                  target="_blank"
-                  rel="noreferrer"
-                >
-                  <PatreonIcon className="h-28 w-28" />
-                </a>
-                <a
-                  href="https://github.com/veganhacktivists"
-                  target="_blank"
-                  rel="noreferrer"
-                >
-                  <GithubIcon className="h-28 w-28" />
-                </a>
-              </div>
+            <div className="border-bg-menu-back-light mt-8 mt-8 flex flex-col border-t pt-8 pb-8">
```

Comment:
```suggestion
            <div className="flex flex-col border-t py-8 mt-8">
```

Plus, can you run Prettier on this? Right now the classes aren't sorted the way they should.